pkg/apiserver/apiserver.go

```diff
@@ -258,7 +258,13 @@ func write(statusCode int, apiVersion string, codec runtime.Codec, object runtim
 		w.WriteHeader(statusCode)
 		writer := w.(io.Writer)
 		if flush {
-			writer = flushwriter.Wrap(w)
+			if _, ok := w.(*hijackTimeoutWriter); ok {
```

Comment:
But more importantly, what goes wrong with these writers that you need to dig inside? Presumably they're wrapped for a reason.